mirrord/agent/src/entrypoint.rs

```diff
@@ -452,30 +452,10 @@ impl ClientConnectionHandler {
                 return Ok(false);
             }
             ClientMessage::PauseTargetRequest(pause) => {
-                match self
-                    .state
-                    .container
-                    .as_ref()
-                    .ok_or(AgentError::PauseAbsentTarget)?
-                    .set_paused(pause)
-                    .await
-                {
-                    Ok(changed) => {
-                        self.respond(DaemonMessage::PauseTarget(
-                            DaemonPauseTarget::PauseResponse {
-                                changed,
-                                container_paused: pause,
-                            },
-                        ))
-                        .await?;
-                    }
-                    Err(e) => {
-                        self.respond(DaemonMessage::LogMessage(LogMessage::error(format!(
-                            "Failed to pause target container: {e:?}"
-                        ))))
-                        .await?;
-                    }
-                }
+                self.respond(DaemonMessage::LogMessage(
```

Comment:
`DaemonMessage::Close` maybe?

Reply:
It'd make a very weird situation where old cli new agent just closes connection..

Reply:
But with a descriptive message :shrug: 
I think it's better than silently (best case that the log is printed to file by intproxy) ignoring pause request